### PR TITLE
Adding 1.8 Samples to ScriptLab playlist

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

File: playlists/excel.yaml [hunk 1]
```diff
@@ -728,6 +737,24 @@
   group: Preview APIs
   api_set:
     ExcelAPI: 1.8
+- id: excel-pivottable-create-and-modify
+  name: Create and modify PivotTable
```
Comment (kbrandl) [inline playlists/excel.yaml]: In the `name` attribute, change "PivotTable" to "Pivot Table"

File: playlists/excel.yaml [hunk 2]
```diff
@@ -728,6 +737,24 @@
...
+  fileName: pivottable-create-and-modify.yaml
+  description: Create and modify a PivotTable
```
Comment (kbrandl) [inline playlists/excel.yaml]: In the `description` attribute, change "PivotTable" to "pivot table"

File: playlists/excel.yaml [hunk 3]
```diff
@@ -728,6 +737,24 @@
...
+  rawUrl: >-
+    https://raw.githubusercontent.com/<ACCOUNT>/<REPO>/<BRANCH>/samples/excel/89-preview-apis/pivottable-create-and-modify.yaml
+  group: Preview APIs
+  api_set:
+    ExcelAPI: 1.8
+- id: excel-pivottable-filters-and-summaries
+  name: Pivot table filters and summaries
+  fileName: pivottable-filters-and-summaries.yaml
+  description: Filter PivotTable data and show different summarizations
```
Comment (kbrandl) [inline playlists/excel.yaml]: In the `description` attribute, change "PivotTable" to "pivot table"

File: playlists/excel.yaml (hunk 1 above)
Comment (kbrandl) [reply playlists/excel.yaml]: So it is!  I confirmed that it is indeed "PivotTable" -- by searching the [Microsoft Style Guide](https://worldready.cloudapp.net/StyleGuide/List) and by looking at the Excel UI itself.

File: playlists/excel.yaml (hunk 2 above)
Comment (kbrandl) [reply playlists/excel.yaml]: (Disregard per my comment above: "PivotTable" is correct.)

File: playlists/excel.yaml [hunk 4]
```diff
@@ -728,6 +737,24 @@
...
+  name: Create and modify PivotTable
+  fileName: pivottable-create-and-modify.yaml
+  description: Create and modify a PivotTable
+  rawUrl: >-
+    https://raw.githubusercontent.com/<ACCOUNT>/<REPO>/<BRANCH>/samples/excel/89-preview-apis/pivottable-create-and-modify.yaml
+  group: Preview APIs
+  api_set:
+    ExcelAPI: 1.8
+- id: excel-pivottable-filters-and-summaries
+  name: Pivot table filters and summaries
```
Comment (kbrandl) [inline playlists/excel.yaml]: To do: In the `name` attribute, change "Pivot table" to "PivotTable".

File: playlists/excel.yaml (hunk 3 above)
Comment (kbrandl) [reply playlists/excel.yaml]: (Disregard per my comment above: "PivotTable" is correct.)

**Comment by kbrandl** [comment]
@AlexJerabek -- IIRC, setting group name to PivotTable should be sufficient.  That being said, once you merge this PR to master, you'll be able to verify that everything looks expected in the `Beta` Script Lab environment (before opening the PR to push the changes to prod).

**Comment by AlexJerabek** [comment]
@kbrandl Oh, I didn't realize there was another step to get this into the public. It there a procedure around pushing to prod?

**Comment by kbrandl** [comment]
Yes, there are more steps to get this into Prod. I will stop by to discuss briefly.

**Pull request merged**